### ISSUE TITLE
CORS on official helios server

### ISSUE THREAD

**Issue by karelbilek** [benadida/helios-server#186]
I want to try independent voting booth using official Helios server

However, the web does not have CORS headers, so you cannot do independent booths. 

(The booth crashes on get_randomness; I can shim it somehow, but it will probably crash elsewhere anyway.)

**Comment by karel-3d** [comment]
It seems CORS are not set at all in the django app.

**Comment by benadida** [comment]
What do you want CORS headers on? The election info URL?

**Comment by karel-3d** [comment]
For independent voting booth :)

**Comment by benadida** [comment]
Right, I'm just trying to make sure that we only open up CORS on limited URLs.. Would you be able to help by identifying which URLs you need available over CORS as you build an independent voting booth?

**Comment by karel-3d** [comment]
I did not try to build my own per se; I tried this directory - https://github.com/benadida/helios-server/tree/master/heliosbooth - on localhost.

**Comment by karelbilek** [comment]
I still think it's not possible to run helios booth independently; or at least I don't know how

**Comment by karelbilek** [comment]
....aaaand yeah, the /meta URL still returns wrong CORS headers when run from localhost.

I *think* it should be possible to have an independent voting booth, which will ensure that there is no hidden JS on server that tracks who votes what.

**Comment by karelbilek** [comment]
it _might_ be possible to get around that by instead of raw HTML, wrapping it in electron and fake the CORS headers. 🤷
